t/tslib/tslib_ubi_8.7.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash -e
+# -----------------------------------------------------------------------------
+#
+# Package	: tslib
+# Version	: 2.6.0
+# Source repo	: https://github.com/microsoft/tslib.git
+# Tested on	: ubi 8.7
+# Language      : TypeScript, JavaScript
+# Travis-Check  : True
+# Script License: Apache License, Version 2 or later
+# Maintainer	: Abhishek Dwivedi <Abhishek.Dwivedi6@ibm.com>
+#
+# Disclaimer: This script has been tested in non root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+PACKAGE_NAME="tslib"
+PACKAGE_VERSION=${1:-"2.6.0"}
+PACKAGE_URL="https://github.com/microsoft/tslib.git"
+export NODE_VERSION=${NODE_VERSION:-v16.20.0}
+HOME_DIR=$PWD
+
+yum install -y git curl
+
+#installing nvm
+cd $HOME_DIR
+curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.3/install.sh | bash
+source "$HOME/.bashrc"
+nvm install "$NODE_VERSION"
+
+cd $HOME_DIR
+git clone $PACKAGE_URL
+cd $PACKAGE_NAME
+git checkout $PACKAGE_VERSION
+
+if ! npm install && npm audit fix && npm audit fix --force; then
+    echo "------------------$PACKAGE_NAME:install_fails-------------------------------------"
+    echo "$PACKAGE_URL $PACKAGE_NAME"
+    echo "$PACKAGE_NAME  |  $PACKAGE_URL | $PACKAGE_VERSION | $OS_NAME | GitHub | Fail |  Install_Fails"
+    exit 1
+fi
+
+if ! node ./test/runTests.js && node ./test/validateModuleExportsMatchCommonJS/index.js ; then
```

Comment:
@AbhiIBM What was reference point for using these commands for install and test steps. Can you please point to there

Reply:
I found the steps to test the package on its CI.yml (https://github.com/microsoft/tslib/blob/2.6.0/.github/workflows/CI.yml#L21)